server/routerlicious/package.json

```diff
@@ -10,6 +10,7 @@
     "build": "npm run policy-check && npm run build:genver && npm run build:compile && npm run lint && npm run build:docs",
     "build:compile": "lerna run build:compile --stream",
     "build:docs": "lerna run build:docs --stream --parallel",
+    "build:docs:ci": "lerna run build:docs:ci --stream --parallel",
```

Comment:
Current pattern is ci:* (instead of *:ci) so that we know the set of all ci command together.

Reply:
fixed in latest iteration